### PR TITLE
Set SSCS Bulk Scanning urls as empty in prod

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-1241

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/demo/cluster-00/bsp/bulk-scan-orchestrator.yaml [hunk 1]
```diff
@@ -22,7 +22,6 @@ spec:
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/bulk-scan/orchestrator:prod-c035f888
       environment:
-        TRANSFORMATION_URL_SSCS: http://sscs-bulk-scan-demo.service.core-compute-demo.internal
```
Comment (nitinprabhu) [inline k8s/demo/cluster-00/bsp/bulk-scan-orchestrator.yaml:25]: why do we need to remove it from demo ?

Comment (nitinprabhu) [reply k8s/demo/cluster-00/bsp/bulk-scan-orchestrator.yaml:25]: or your updating processor values yaml ?

Comment (aliveni) [reply k8s/demo/cluster-00/bsp/bulk-scan-orchestrator.yaml:25]: we don't need it after this change https://github.com/hmcts/bulk-scan-processor/pull/1328

Comment (aliveni) [reply k8s/demo/cluster-00/bsp/bulk-scan-orchestrator.yaml:25]: I thought of raising a separate PR for version. can do in this PR

Comment (nitinprabhu) [reply k8s/demo/cluster-00/bsp/bulk-scan-orchestrator.yaml:25]: so then we need chart bump in this PR. So first we need to merge PR 1328 and then this should have version bump change.
Also worth checking when is SSCS going to prod as we may just remove empty strings for prod

Comment (nitinprabhu) [reply k8s/demo/cluster-00/bsp/bulk-scan-orchestrator.yaml:25]: same goes for orchestrator

**Pull request merged**